docs/platform/howto/saml/setup-saml-google.rst

```diff
@@ -0,0 +1,84 @@
+Set up SAML with Google
+=======================
+
+This article explains how to set up SAML with Google for an organization in Aiven. For more information on SAML and instructions for other identity providers, see the :doc:`Set up SAML authentication </docs/platform/howto/saml/saml-authentication>` article.
+
+Prerequisite steps in Aiven Console
+------------------------------------
+
+#. In the organization, click **Admin**.
+
+#. Select **Authentication**.
+
+#. Click **Add authentication method**.
+
+#. Enter a name and select SAML. You can also select the teams that users will be added to when they sign up or log in through this authentication method.
+
+You are shown two parameters needed to set up the SAML authentication in Google:
+
+* Metadata URL
+* ACS URL
+
+Configure SAML on Google
+------------------------
+
+1. Log in to Google Admin console
```

Comment:
```suggestion
1. Log in to Google Admin console.
```

Reply:
Sorry, missed this on the first pass, but all instructions should have a period at the end.

Reply:
Added the period at the end.